lib/main_containers.pm

```diff
@@ -89,7 +89,9 @@ sub load_host_tests_podman {
         load_image_test($run_args) unless is_public_cloud || is_alp;
         load_3rd_party_image_test($run_args);
         loadtest 'containers/podman_pods';
-        loadtest 'containers/podman_network';
+        loadtest('containers/podman_network', name => 'podman_network_cni');
+        loadtest 'containers/podman_netavark';
+        loadtest('containers/podman_network', name => 'podman_network_netavark');
```

Comment:
I don't understand why you load the `podman_network` test twice only with different name. 😞

Reply:
We are testing 2 network backends.

Reply:
Yes, we agreed to test also regular network tests with both backends.
There was the option of doing the backend switch inside `podman_network` itself, but this looks a bit more transparent for the test module.

There is one thing I don't like though. You are doing this schedule in ALL cases. We need to find a way to only do repeat this module only when `package_version_cmp($podman_version, '4.4.2') < 0`.  Otherwise, we will have a "false" `podman_network_netavark` coverage in maintenance.

Reply:
> There is one thing I don't like though. You are doing this schedule in ALL cases. We need to find a way to only do repeat this module only when `package_version_cmp($podman_version, '4.4.2') < 0`. Otherwise, we will have a "false" `podman_network_netavark` coverage in maintenance.

The problem is that the SUT is not booted at the time of test module schedule. Therefore, I have added the podman version check in the test module on L#44

Reply:
Ok, then we need to think a way how to fix this, it doesn't make sense to schedule the test twice in every case.